HW1/P2/P2.txt

```diff
@@ -1 +1,4 @@
 # Your discussion here
```

Comment:
Good point on the tradeoff of improving the load balancing, it turns out nothing comes for free with spark. When I was implementing the problem for the first time, I didn't realize how costly shuffles can be.